index.test.js

```diff
@@ -128,56 +145,52 @@ describe('Update Pull Request', () => {
       expect(action.getStoryIds(pullRequest)).toEqual(['2']);
     });
 
-    test('should exit if no sc- id in PR title or branchName', () => {
+    test('should return an empty array if no sc- id in PR title or branchName', () => {
       const pullRequest = {
         title: 'I have nothing related to sc- ids in my title',
         head: { ref: 'i-dont-have-a-sc-id-in-here' },
       };
 
-      action.getStoryIds(pullRequest);
-      expect(core.setFailed).toHaveBeenCalledTimes(1);
+      const result = action.getStoryIds(pullRequest);
+      expect(result).toEqual([]);
     });
   });
 
-  describe('getShortcutStory', () => {
-    let chMock;
-    let client;
-    let stories;
-    let storyIds;
+  describe('fetchStoryAndUpdatePr', () => {
```

Comment:
Replaced with a medium sized test of the core functionality to make sure things are wired up correctly